<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultimediaPlayerWidget</class>
 <widget class="QWidget" name="MultimediaPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multimedia Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(91, 91, 91, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/Hit_Radio_FFH_-_Logo_Neu.svg.png</normaloff>:/Images/Hit_Radio_FFH_-_Logo_Neu.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/SWR3_Logo.svg.png</normaloff>:/Images/SWR3_Logo.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/SWR1.svg.png</normaloff>:/Images/SWR1.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/Bayern3_logo_2015.svg.png</normaloff>:/Images/Bayern3_logo_2015.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/Hr3-Logo_2015.svg.png</normaloff>:/Images/Hr3-Logo_2015.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/Home.png</normaloff>:/Images/Home.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>381</width>
      <height>40</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	border: 1px solid #bbb;
	background: white;
	height: 10px;
	border-radius: 5px;
}

QSlider::add-page:horizontal {
	background: #fff;
	border: 1px solid #777;
	height: 10px;
	border-radius: 4px;
}

QSlider::sub-page:horizontal {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
   		stop: 0 rgba(255, 255, 255, 255), stop: 1 rgba(129, 0, 0, 255));
	background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
   		stop: 0 rgba(255, 255, 255, 255), stop: 1 rgba(129, 0, 0, 255));
	border: 1px solid #777;
	height: 10px;
	border-radius: 4px;
}

QSlider::handle:horizontal {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    	stop:0 #eee, stop:1 #ccc);
	border: 2px solid #777;
	width: 16px;
	height: 20px;
	line-height: 20px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 10px;
}

QSlider::handle:horizontal:hover {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    	stop:0 rgba(72, 68, 71, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid #777;
	width: 16px;
	height: 20px;
	line-height: 20px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 10px;
}

QSlider::sub-page:horizontal:disabled {
	background: #bbb;
	border-color: #999;
}

QSlider::add-page:horizontal:disabled {
	background: #eee;
	border-color: #999;
}

QSlider::handle:horizontal:disabled {
	background: #eee;
	border: 1px solid #aaa;
	border-radius: 4px;
}

QSlider{
	background-color: transparent;
}






</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="addfav">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Favoriten hinzuf√ºgen</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/add_favourite.png</normaloff>:/Images/add_favourite.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>5</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Radio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>160</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_date_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>5</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Favoriten:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>480</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MuteButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border-width: 5px;
	border-radius: 10px;
}
QPushButton:pressed {
	border-color: rgb(0, 0, 0);
	border-width : 1.2px;
	border-style:inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/speaker_muted.png</normaloff>:/Images/speaker_muted.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border-width: 5px;
	border-radius: 10px;
}
QPushButton:pressed {
	border-color: rgb(0, 0, 0);
	border-width : 1.2px;
	border-style:inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/trashcan_icon.png</normaloff>:/Images/trashcan_icon.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_fav">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Navigation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/icon_navigation.png</normaloff>:/Images/icon_navigation.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>MEN√ú</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Media">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/music_icon_2.png</normaloff>:/Images/music_icon_2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Phone">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/phone_icon_1.png</normaloff>:/Images/phone_icon_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Images/radio_icon_1.png</normaloff>:/Images/radio_icon_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
